mypy/newsemanal/typeanal.py

```diff
@@ -210,6 +210,7 @@ def visit_unbound_type_nonoptional(self, t: UnboundType, defining_literal: bool)
                 an_args = self.anal_array(t.args)
                 disallow_any = self.options.disallow_any_generics and not self.is_typeshed_stub
                 res = expand_type_alias(target, all_vars, an_args, self.fail, node.no_args, t,
+                                        unexpanded_type=t,
```

Comment:
This change looks unrelated, or am I missing something?

Reply:
`expand_type_alias` is used by the type checker and switching the checker to using the new analyzer required some tweaks